templates/apps/sumo-aws-apps/sumologic-app-utils/src/awsresource.py

```diff
@@ -1544,33 +1814,33 @@ def create_members(self,guardduty_client, detector_id: str, accounts: list):
         number_of_create_members_calls = math.ceil(len(accounts) / 50)
         for api_call_number in range(0, number_of_create_members_calls):
             account_details = accounts[api_call_number * 50 : (api_call_number * 50) + 50]
-            logger.info(f"Calling create_member, api_call_number {api_call_number} with detector_id: {detector_id}")
-            logger.info(f"Create member account_details: {account_details}, account_details length: {len(account_details)}")
+            #logger.info(f"Calling create_member, api_call_number {api_call_number} with detector_id: {detector_id}")
+            #logger.info(f"Create member account_details: {account_details}, account_details length: {len(account_details)}")
             create_members_response = guardduty_client.create_members(DetectorId=detector_id, AccountDetails=account_details) 
             if "UnprocessedAccounts" in create_members_response and create_members_response["UnprocessedAccounts"]:
                 unprocessed = True
                 retry_count = 0
                 unprocessed_accounts = []
-                logger.info(f"Retry number; {retry_count} for unprocessed accounts")
+                #logger.info(f"Retry number; {retry_count} for unprocessed accounts")
                 logger.info(f"Sleeping for {self.SLEEP_SECONDS} before retry")                
                 while unprocessed:
                     retry_count += 1
-                    logger.info(f"Unprocessed Accounts: {create_members_response['UnprocessedAccounts']}")
+                    l#ogger.info(f"Unprocessed Accounts: {create_members_response['UnprocessedAccounts']}")
                     remaining_accounts = self.get_unprocessed_account_details(create_members_response, accounts)
 
                 if len(remaining_accounts) > 0:
-                    logger.info("Remaining accounts found during create members")
-                    logger.info(f"Calling create_member, api_call_number {api_call_number} with detector_id: {detector_id}")
-                    logger.info(f"Create member account_details: {remaining_accounts}, remaining_accounts length: {len(remaining_accounts)}")
+                    #logger.info("Remaining accounts found during create members")
+                    #logger.info(f"Calling create_member, api_call_number {api_call_number} with detector_id: {detector_id}")
+                    #logger.info(f"Create member account_details: {remaining_accounts}, remaining_accounts length: {len(remaining_accounts)}")
```

Comment:
these lines do not contain sensitive info may be we can put some of them in debug.

Reply:
okay